package.json

```diff
@@ -1,12 +1,28 @@
 {
+  "name": "polkadot",
```

Comment:
```suggestion
  "name": "polkadot-apps",
```

Reply:
Earlier I left it here because that field is related to the electron app name but for now, I also add a field in electron build config for that, so field 'name' is no longer related to the electron build process.